test/functional/ui/popupmenu_spec.lua

```diff
@@ -2488,6 +2538,11 @@ describe('builtin popupmenu', function()
         end
         feed('<C-E><ESC>')
       end)
+
+      it('bufnr of popup floating window does not changed', function()
+        feed('S<C-x><C-o><C-N>')
```

Comment:
maybe mutliple `<c-n><c-n>` to emphasize that new info windows are being displayed?
```suggestion
        feed('S<C-x><C-o><C-N><C-N>')
```

can this test also assert that multiple info windows were displayed, somehow?

Reply:
the cot use default value that mean when trigger omni it select first item <C-N> select two so just one `<C-N>` is enough here otherwise it will select 3 item which use `nvim__complete_set` this will generate a new buffer . because preview buffer is `wipe` when floating window is hide the buffer is removed. calling `nvim__complete_set` will show flaoting window with a new buffer. so now is fine